specs/protocol/jovian/bridges.md

```diff
@@ -2,12 +2,311 @@
+
+## Legacy Withdrawal Support
+
+The L1CGTBridge provides functionality to handle legacy withdrawals initiated before CGT migration. These functions use a trusted storage root to prove and finalize withdrawals containing native asset transfers (`_tx.value > 0`) that cannot be processed through the standard OptimismPortal after migration.
+
+## L1CGTBridge
+
+The L1CGTBridge contract is deployed individually for each L2 chain using Custom Gas Token mode. It manages the L1 side of CGT bridging operations.
+
+### Interface
+
+```solidity
+interface IL1CGTBridge {
+    // Events
+    event CGTBridgeInitiated(address indexed from, address indexed to, uint256 amount, bytes extraData);
+    event CGTBridgeFinalized(address indexed from, address indexed to, uint256 amount, bytes extraData);
+    event WithdrawalProven(bytes32 indexed withdrawalHash, address indexed from, address indexed to);
+    event WithdrawalProvenExtension1(bytes32 indexed withdrawalHash, address indexed proofSubmitter);
+    event WithdrawalFinalized(bytes32 indexed withdrawalHash, bool success);
+
+    // Core bridge functions
+    function bridgeCGT(uint256 _amount, uint32 _minGasLimit, bytes calldata _extraData) external;
+    function bridgeCGTTo(address _to, uint256 _amount, uint32 _minGasLimit, bytes calldata _extraData) external;
+    function finalizeBridgeCGT(address _from, address _to, uint256 _amount, bytes calldata _extraData) external;
+
+    // Legacy withdrawal functions (for CGT migration)
+    function legacyProveWithdrawalTransaction(Types.WithdrawalTransaction memory _tx, bytes[] calldata _withdrawalProof) external;
+    function legacyFinalizeWithdrawalTransaction(Types.WithdrawalTransaction memory _tx) external;
+
+    // Trusted state management (migration only)
+    function setTrustedStateOnce(bytes32 trustedMessagePasserStorageRoot) external;
+
+    // Bridge management
+    function enableDeposits() external;
+    function disableDeposits() external;
+    function enableWithdrawals() external;
+    function disableWithdrawals() external;
+
+    // Configuration getters
+    function cgtToken() external view returns (address);
+    function otherBridge() external view returns (address);
+    function messenger() external view returns (address);
+    function trustedMessagePasserStorageRoot() external view returns (bytes32);
+}
+```
+
+### Functions
+
+#### `bridgeCGT`
+
+Initiates a CGT transfer from L1 to L2 for the caller.
+
+```solidity
+function bridgeCGT(uint256 _amount, uint32 _minGasLimit, bytes calldata _extraData) external
+```
+
+- MUST transfer `_amount` of CGT ERC20 tokens from `msg.sender` to the bridge contract
+- MUST send a message to L2CGTBridge via CrossDomainMessenger to mint equivalent native assets to `msg.sender`
+- MUST emit `CGTBridgeInitiated` event
+- MUST revert if `_amount` is zero
+- MUST revert if caller has insufficient token balance or allowance
+
+#### `bridgeCGTTo`
```

Comment:
Change to bridgeCGT
Bridges the specified amount of the designated ERC20 from L1 to L2 to a specified recipient.